src/glue.cpp

```diff
@@ -20,16 +20,62 @@ namespace nodemg {
 
 Napi::Value MgStringToNapiString(Napi::Env env, const mg_string *input_string) {
   Napi::EscapableHandleScope scope(env);
-
```

Comment:
Dead removed line? Is this intentional?

Reply:
Yes, because I thought these empty lines made sense in the first place, but I don't think so anymore...